docs/assets/2.14-2.15-cleanup-telemetry-fluent-bit.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
```

Comment:
Rename this file. It cleans up more than just fluent-bit.

Reply:
renamed